lib/proto/google/api/http_pb.js

```
@@ -0,0 +1,1012 @@
+ * @param {!jspb.BinaryReader} reader The BinaryReader to use.
+ * @return {!proto.google.api.Http}
+ */
+proto.google.api.Http.deserializeBinaryFromReader = function(msg, reader) {
+  while (reader.nextField()) {
+    if (reader.isEndGroup()) {
+      break;
+    }
+    var field = reader.getFieldNumber();
+    switch (field) {
+    case 1:
+      var value = new proto.google.api.HttpRule;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)